### PR TITLE
Rename Quest Module To OpenSRP

### PR DESCRIPTION
**Pull request by ndegwamartin**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Renaming The Quest Module to OpenSRP 2

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [x] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/opensrp/build.gradle.kts [hunk 1]
```diff
@@ -135,7 +160,6 @@ android {
   productFlavors {
     create("opensrp") {
       dimension = "apps"
-      applicationIdSuffix = ".opensrp"
       manifestPlaceholders["appLabel"] = "OpenS+"
```
[the remark this answers is not in the document]
Comment (ellykits) [reply android/opensrp/build.gradle.kts:163]: cc @pld

**Comment by ellykits** [comment]
Attempting to resolve the dropped coverage (caused by missing coverage report for the quest module).

**Comment by ellykits** [comment]
Removed requirement check for `quest-tests`. Will add `opensrp-tests` once this PR is merged to allow the merging of other PRs.

**Comment by ndegwamartin** [comment]
As discussed closing this PR in favor of https://github.com/opensrp/fhircore/pull/2464

**Pull request closed without merging**